src/java.base/share/classes/java/security/BasicPermission.java

```diff
@@ -491,10 +495,14 @@ public boolean implies(Permission permission) {
     /**
      * @serialData Default fields.
      */
```

Comment:
FWIW, this doc comment will be ignored, because it will be superseded by the new comment on line 499.  At some point doen the road, you may get a warning from javac about an ignored doc comment.

Reply:
Ok

Reply:
Removed after received similar later comments.